election-api-typescript/src/resultsController.ts

```diff
@@ -0,0 +1,24 @@
+import resultStore from "./resultsService";
+import { Result } from "./types";
+
+const store = resultStore();
+
+const getResult = (id: number): Result | undefined => {
+  return store.getResult(id);
+};
+
+// TODO: Ask someone why this returns empty object instead of result and add type
+const newResult = (result: Result) => {
+  store.newResult(result);
+  return {};
+};
```

Comment:
It is not clear why this returns an empty object instead of the `result`. Any ideas?

Reply:
I don't think there is a good reason to be honest. There are lots of things you _could_ return that would be sensible. I'd probably either return nothing (i.e. `null`) and rely on the status code indicating that the write was successful or something with a message saying that the result was created successfully.